app/assets/javascripts/revised/pages/welcome/recovery_stories.coffee

```diff
@@ -1,9 +1,10 @@
 class BikeIndex.RecoveryStories extends BikeIndex
   constructor: ->
-    $recovery_stories_containers = $('#recovery-stories-container').find('.recovery-stories-container')
```

Comment:
(This query string is incorrect)

Reply:
Just to make sure - it's now the correct query string, with the class, and this is ready?

Reply:
Yup, thanks—this is the old line. It should be `.recovery-stories-container`.